demos/browser/app/meetingV2/meetingV2.html

```diff
@@ -149,18 +149,12 @@ <h5 class="modal-title" id="additional-options-modal-label">Additional Options</
             <label for="priority-downlink-policy" class="form-check-label">Use Priority-Based Downlink
               Policy</label>
           </div>
-          <lable id="priority-downlink-policy-preset-title" for="priority-downlink-policy-preset" style="display: none; padding-top:5px">Policy Preset</lable>
-          <select id="priority-downlink-policy-preset" class="form-select" style="width:100%; display: none;">
-            <option value="default" selected>Default</option>
```

Comment:
Do we want to remove it now as we still have not really removed it from our code yet we may want to keep it in order to test?

Reply:
This isn't currently in any QA regression tests AFAIK, so I think the benefits of not accidentally confusing a builder (or leading them to not use the latest SSNA) outweigh the benefits of being able to smoke test an infrequently used feature that we wouldn't fix issues for anyways (since we would just tell builders to use SSNA :P).